actions/build_deploy_pex/action.yml

```diff
@@ -20,12 +20,10 @@ inputs:
 
 runs:
   using: "composite"
+
   steps:
-    - name: Checkout action repo
-      uses: actions/checkout@v3
-      with:
-        repository: dagster-io/dagster-cloud-action
-        path: $GITHUB_WORKSPACE/action-repo
+    - run: echo "ACTION_REPO=$GITHUB_ACTION_PATH/../../" >> $GITHUB_ENV
```

Comment:
This change makes it possible to test this PR branch correctly, without having to hack anything.